local/rest_api_gcbm/README.md

```diff
@@ -1,38 +1,88 @@
+  ```
+4. Create a container over the `gcbm-api` image and start it : 
+ ```
+    docker run --rm -p 8080:8080 gcbm-api
+  ```
 
-```bash
-docker-compose down
-```
+Navigate to http://localhost:8080/ in the browser to test the various endpoints available.
 
 Currently the REST API has the following endpoints available for access:-
  
 |      Endpoint     | Functionality | Method | 
 | :----------------: | :----------------: | :----------------: | 
-|   **\help\all**    | This endpoint produces a help message with information on all options for moja.CLI | `GET`
-|   **\help\arg**    |  This endpoint produces a help message with information on option arg for moja.CLI. | `GET`
-|   **\gcbm\new**    |  This endpoint creates a new directory to store input and output files of the simulation. Parameter to be passed in the body is the title of the new simulation or default value simulation will be used.   | `POST` |
-|  **\gcbm\upload**  | This endpoint is used to upload the input files: config_files, input and database. Remember to upload the files in the body from the GCBM folder which is available in the root of this repository for setup and use. A directory /input will be created to store the specified files. | `POST` |
-| **\gcbm\dynamic**  |  This endpoint runs simulation in a thread and generates an output zip file in /output directory. The process may take a while. Parameter is the title of the simulation. | `POST` |
-|  **\gcbm\status**  | This endpoint is employed to check the status of the simulation. It sends a message 'Output is ready to download' to notify that the output zip file is generated. Parameter is the title of the simulation. | `POST`
-|  **\gcbm\download**  | This endpoint is used to download the output zip file. Parameter is the title of the simulation. | `POST`
-|  **\gcbm\list**  | This endpoint retrieves the complete list of simulations that are created using /new.  | `GET`
+|   **/help/all**    | This endpoint produces a help message with information on all options for moja.CLI | `GET`
+|   **/help/arg**    |  This endpoint produces a help message with information on option arg for moja.CLI. | `GET`
+|   **/gcbm/new**    |  This endpoint creates a new directory to store input and output files of the simulation. Parameter to be passed in the body is the title of the new simulation or default value simulation will be used.   | `POST` |
+|  **/gcbm/upload**  | This endpoint is used to upload the input files: config_files, input and database. Remember to upload the files in the body from the GCBM folder which is available in the root of this repository for setup and use. A directory /input will be created to store the specified files. | `POST` |
+| **/gcbm/dynamic**  |  This endpoint runs simulation in a thread and generates an output zip file in /output directory. The process may take a while. Parameter is the title of the simulation. | `POST` |
```

Comment:
Can you please file an issue to rename this endpoint `/gcbm/run`? I don't know why it's called dynamic

Reply:
Will do it.